### PR TITLE
Added Selection Sort [Ruby]

### PR DESCRIPTION
**Pull request by 15BCS047**
Added Selection_Sort.rb
#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [ ] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [ ] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [ ] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

**Comment by singhpratyush** [comment]
Hi! Thanks for showing interest in contributing to this repository.

Please make sure that you have ticked the points mentioned in PR description correctly.

Thanks again!

File: Selection_Sort/Selection_Sort.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+def SelectionSort(a)# passing array in SelectionSort fuction 
+  n = a.length - 1# n is a variable which holds the  length of array which we can be find  by using inbuilt 'length' function
```
Comment (mohitkyadav) [inline Selection_Sort/Selection_Sort.rb]: `# Comment`
`code`
Please follow this convention.

File: Selection_Sort/Selection_Sort.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# passing array in SelectionSort fuction 
+def SelectionSort(a)
```
Comment (mohitkyadav) [inline Selection_Sort/Selection_Sort.rb]: Use snake case for function name.
Also put a comment containing time and space complexity of the algorithm.

Comment (15BCS049) [reply Selection_Sort/Selection_Sort.rb]: I will change it as currently exam is going on

**Comment by aashutoshrathi** [comment]
@15BCS047 Just wait for a while we are adding Ruby Bear for syntax checks.

**Pull request closed without merging**